modules/cluster-logging-deploy-cli.adoc

```diff
@@ -0,0 +1,406 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-cli_{context}"]
+= Installing cluster logging using the CLI
+
+[role="_abstract"]
+You can use the {product-title} CLI to install the Elasticsearch and Cluster Logging operators.
```

Comment:
`{product-title}` should probably be renamed to just `oc`. Because if these docs are going to be reused for ROSA, we probably don't want to confuse customers by referring to the `rosa` CLI

Reply:
Fixed.